packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -135,16 +135,13 @@
 	}
 }
 
-.edit-site-global-styles__shadow-editor-panel {
+.edit-site-global-styles__shadow-editor__dropdown-content {
 	width: 280px;
-	padding: $grid-unit-20;
```

Comment:
This padding is equivalent to `paddingSize="medium"` on `DropdownContentWrapper`.